### DISCUSSION TITLE
Fedora dependency update required for compiling.md

### DISCUSSION THREAD

**Discussion by stevie2sheds** [jamulussoftware/jamulus#3365] [Documentation]
For compiling on Fedora one of the dependencies, qt5-qtmultimedia-devel, is missing from the list on compiling.md. The Debian list still works like a charm.

**Comment by ann0see** [comment]
Please open an issue on that and provide a new package if you know it.

**Comment by ann0see** [comment]
Continues at https://github.com/jamulussoftware/jamulus/issues/3366
